### PR TITLE
Feat/unified notification on proposes loaded massively

### PR DESCRIPTION
**Pull request by ivan-mr**
#### :tophat: What? Why?
Replace notifying every proposal or answer to followers and notify the follower once time reporting all imported elements

#### :pushpin: Related Issues
- Related to https://3.basecamp.com/4186608/buckets/11242950/card_tables/cards/7597830793

#### :clipboard: Subtasks
- [ ] Add documentation regarding the feature 
- [ ] Add/modify seeds
- [x] Add tests
- [ ] Another subtask

### :camera: Screenshots (optional)
Screenshot of email send notifying proposals import
image

Screenshot of email send notifying answers proposals import
image

### CODE REVIEW

File: app/decorators/decidim/admin/import/importer_decorator.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      recipients.each do |recipient|
+        next unless ["all", "followed-only"].include?(recipient.notification_types)
+
+        ProposalsMailer.notify_massive_import(collection, recipient).deliver_now
+      end
+    when :answers
+      recipients.each do |recipient|
+        next unless ["all", "followed-only"].include?(recipient.notification_types)
+
+        ProposalsAnswersMailer.notify_massive_import(collection, recipient).deliver_now
```
Comment (laurajaime) [inline app/decorators/decidim/admin/import/importer_decorator.rb:26]: I think is better `deliver_later`, what do you think?

File: app/mailers/proposals_answers_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+class ProposalsAnswersMailer < ApplicationMailer
+  include Decidim::SanitizeHelper
+  include Decidim::ComponentPathHelper
+
+  layout "decidim/mailer"
+
+  def notify_massive_import(answers_collection, user)
+    @organization = user.organization
+    @participatory_space_title = decidim_sanitize_translated(answers_collection.first.participatory_space.title) if answers_collection.first.present?
```
Comment (laurajaime) [inline app/mailers/proposals_answers_mailer.rb:11]: extract `answers_collection.first` to a variable and avoid doing multiple times.

File: app/mailers/proposals_answers_mailer.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    @user = user
+    @component_url = manage_component_url(answers_collection.first.component)
+    I18n.locale = user.locale if user.locale.present?
+
+    subject = I18n.t("proposals_answers_publisheds_for_space.email_subject", scope: "decidim.events.proposals", participatory_space_title: @participatory_space_title)
```
Comment (laurajaime) [inline app/mailers/proposals_answers_mailer.rb:16]: what is publisheds? 🤔  I think is incorrect..

File: app/mailers/proposals_mailer.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+class ProposalsMailer < ApplicationMailer
+  include Decidim::SanitizeHelper
+  include Decidim::ComponentPathHelper
+
+  layout "decidim/mailer"
+
+  def notify_massive_import(proposals_collection, user)
+    @organization = user.organization
+    @participatory_space_title = decidim_sanitize_translated(proposals_collection.first.participatory_space.title) if proposals_collection.first.present?
```
Comment (laurajaime) [inline app/mailers/proposals_mailer.rb:11]: extract `proposals_collection.first` to a variable and avoid doing multiple times.

File: app/mailers/proposals_mailer.rb [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    @user = user
+    @component_url = manage_component_url(proposals_collection.first.component)
+    I18n.locale = user.locale if user.locale.present?
+
+    subject = I18n.t("proposals_publisheds_for_space.email_subject", scope: "decidim.events.proposals", participatory_space_title: @participatory_space_title)
```
Comment (laurajaime) [inline app/mailers/proposals_mailer.rb:16]: what is publisheds? 🤔 I think is incorrect..

File: app/views/proposals_answers_mailer/notify_massive_import.html.erb [hunk 6]
```diff
@@ -0,0 +1,4 @@
+<p class="email-greeting"><%= I18n.t("decidim.events.email_event.email_greeting", user_name: @user.name) %></p>
+<p class="email-instructions"><%= I18n.t("decidim.events.proposals.proposals_answers_publisheds_for_space.email_intro", participatory_space_title: @participatory_space_title).html_safe %></p>
```
Comment (laurajaime) [inline app/views/proposals_answers_mailer/notify_massive_import.html.erb:2]: review the publisheds word please

File: decidim-process-extended/config/locales/ca.yml [hunk 7]
```diff
@@ -47,6 +47,9 @@ ca:
             left: Restant
             title: Té %{limit} suports a distribuir
             votes: Suports
+    events:
+      publish_proposals_event:
+        email_intro: S'han afegit noves propostes a l'espai "%{resource_title}". Pots veure les noves propostes seguint <a href="%{resource_url}">aquest enllaç</a>.
```
Comment (laurajaime) [inline decidim-process-extended/config/locales/ca.yml:52]: is missing in en and oc, right?

File: spec/decorators/decidim/admin/import/importer_decorator_spec.rb [hunk 8]
```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+require "decidim/proposals/import/proposal_creator_decorator"
+# require 'spec/shared/proposal_importer_examples'
```
Comment (laurajaime) [inline spec/decorators/decidim/admin/import/importer_decorator_spec.rb:5]: remove?

**Review by laurajaime — COMMENTED** [review]
Good! some comments to review

File: spec/decorators/decidim/admin/import/importer_decorator_spec.rb (hunk 8 above)
Comment (ivan-mr) [reply spec/decorators/decidim/admin/import/importer_decorator_spec.rb]: Removed! thanks

File: app/decorators/decidim/admin/import/importer_decorator.rb (hunk 1 above)
Comment (ivan-mr) [reply app/decorators/decidim/admin/import/importer_decorator.rb]: Ok, I put deliver_now to develop properly, but now is changed to deliver_later.
Thanks!

File: app/mailers/proposals_answers_mailer.rb (hunk 2 above)
Comment (ivan-mr) [reply app/mailers/proposals_answers_mailer.rb]: Yes, you're right. Thanks!

File: app/mailers/proposals_mailer.rb (hunk 4 above)
Comment (ivan-mr) [reply app/mailers/proposals_mailer.rb]: Done like previous suggestion, thanks!

File: app/mailers/proposals_mailer.rb (hunk 5 above)
Comment (ivan-mr) [reply app/mailers/proposals_mailer.rb]: Done like previous suggestion, thanks!

File: app/views/proposals_answers_mailer/notify_massive_import.html.erb (hunk 6 above)
Comment (ivan-mr) [reply app/views/proposals_answers_mailer/notify_massive_import.html.erb]: Replacing key like previous suggestion. Thanks!

File: decidim-process-extended/config/locales/ca.yml (hunk 7 above)
Comment (ivan-mr) [reply decidim-process-extended/config/locales/ca.yml:52]: I put the translations for the other locales. Thanks!

File: spec/decorators/decidim/proposals/import/proposal_creator_decorator_spec.rb [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+      "id" => "101",
+      category: category,
+      scope: scope,
+      :"title/en" => Faker::Lorem.sentence,
+      :"body/en" => Faker::Lorem.paragraph(sentence_count: 3),
+      address: "#{Faker::Address.street_name}, #{Faker::Address.city}",
+      latitude: Faker::Address.latitude,
+      longitude: Faker::Address.longitude,
+      component: component,
+      published_at: moment
```
Comment (laurajaime) [inline spec/decorators/decidim/proposals/import/proposal_creator_decorator_spec.rb:22]: can you unify syntax? there are `key: value` and `key => value` syntax, but arrow is an obsolete way.  And why two ids?

**Pull request merged**